src/scripts/main.js

```diff
@@ -1,3 +1,87 @@
 'use strict';
 
-// write code here
+const field = document.querySelector('.container');
+const tBody = document.querySelector('tbody');
+const addRowBtn = document.querySelector('.append-row');
+const removeRowBtn = document.querySelector('.remove-row');
+const addColBtn = document.querySelector('.append-column');
+const removeColBtn = document.querySelector('.remove-column');
+
+function addRow(table) {
+  table.firstElementChild.before(table.firstElementChild.cloneNode(true));
+}
+
+function removeRow(table) {
+  table.lastElementChild.remove();
+}
+
+function addColumn(table) {
+  Array.from(table.rows).forEach(row => row
+    .lastElementChild
+    .after(row.lastElementChild.cloneNode(true)));
```

Comment:
improve it everywhere, for better readability
```suggestion
    .after(row.lastElementChild.cloneNode(true))
  );
```